setup.cfg

```diff
@@ -26,7 +26,7 @@ packages = find:
 install_requires=
     gitpython>3
     dulwich>=0.20.34
-    pygit2>=1.5.0
+    pygit2>=1.6.0
```

Comment:
I think we should make this 1.7.2 (see https://github.com/iterative/scmrepo/pull/50)